#RMSSD: Root Mean Square of Successive Differences 
time stamp
uint8 heart_rate
float32 rmssd